README.md

```diff
@@ -90,6 +90,7 @@ the same configuration page.
 * S3 Bucket Name: Name of the S3 Bucket to use to store artifacts.
 * S3 Bucket Region: Region to use to generate the URLs to get/put artifacts, by default it is autodetected.
 * Base Prefix: Prefix to use for files and folders inside the S3 Bucket, if the prefix is a folder should be end with `/`.
+* Use Accelerated Endpoint: Use accelerated endpoint for the S3 Bucket.
```

Comment:
I would link to https://docs.aws.amazon.com/AmazonS3/latest/dev/transfer-acceleration.html to explain what acceleration means

Reply:
done